tests/unit/core/statistics/test_constraints.py

```diff
@@ -401,3 +408,49 @@ def test_max_between_constraint_invalid():
         maxBetweenConstraint(lower_value="2", upper_value=2)
     with pytest.raises(TypeError):
         maxBetweenConstraint(lower_field="max", upper_field=2)
+
+
+def test_column_values_in_set_constraint(df_lending_club, local_config_path):
+    cvisc = columnValuesInSetConstraint(value_set={2, 5, 8})
+    ltc = ValueConstraint(Op.LT, 1)
+    dc = DatasetConstraints(None, value_constraints={"annual_inc": [cvisc, ltc]})
+    config = load_config(local_config_path)
+    session = session_from_config(config)
+    profile = session.log_dataframe(df_lending_club, "test.data", constraints=dc)
+    session.close()
+    report = dc.report()
+
+    assert report[0][1][0][1] == len(df_lending_club)
```

Comment:
Not your fault, but our API for the report object is not easy to read, can you add a comment on why the length of the lending club data is what you are checking here?

Reply:
Sure I'll add it in the tests. I check for the length of the dataframe to validate if all the rows were reported.